src/fides/api/schemas/connection_configuration/connection_secrets_fides.py

```diff
@@ -10,8 +12,8 @@ class FidesConnectorSchema(ConnectionConfigSecretsSchema):
     """Schema to validate the secrets needed to connect to a remote Fides"""
 
     uri: str
-    username: str
-    password: str
+    username: str = Field(sensitive=True)
+    password: str = Field(sensitive=True)
```

Comment:
We can connect to remote fides instances? Is this brand new - I can't find it in the fides 2.14.0 admin-ui add connector page?